src/dev/build/tasks/create_cdn_assets_task.ts

```diff
@@ -31,9 +33,19 @@ export const CreateCdnAssets: Task = {
     await del(assets);
     await mkdirp(assets);
 
-    // Plugins
-
     const plugins = globby.sync([`${buildSource}/node_modules/@kbn/**/*/kibana.jsonc`]);
+
+    // translation files
+    const pluginPaths = plugins.map((plugin) => resolve(dirname(plugin)));
+    for (const locale of supportedLocale) {
+      const translationFileContent = await generateTranslationFile(locale, pluginPaths);
```

Comment:
Is this deterministic?  Any reason not to copy over the files from `node_modules/@kbn/translations-plugin/translations`?

Reply:
the actual translation files sent to the server aren't following the same format than the ones in `node_modules/@kbn/translations-plugin/translations` (even if in practice, the files from the `translations` plugin is the only "source" to generate those final ones).